src/rust/bitbox02-rust/src/hww/api.rs

```diff
@@ -108,33 +114,54 @@ fn can_call(request: &Request) -> bool {
     match request {
         // Deprecated call, last used in v1.0.0.
         Request::PerformAttestation(_) => false,
-        Request::DeviceInfo(_) => true,
-        Request::Reboot(_) => true,
-        Request::DeviceName(_) => true,
-        Request::DeviceLanguage(_) => true,
-        Request::CheckSdcard(_) => true,
-        Request::InsertRemoveSdcard(_) => true,
-        Request::ListBackups(_) => true,
+        Request::DeviceInfo(_) => matches!(
```

Comment:
Using some helper function to check for the state condition could help readability and reduce code redundancy, wdyt?

Reply:
Adding helper vars didn't work to help readability, but I grouped the messages that have the same checks, which seems a bit better. Please check.